metadata.yaml

```diff
@@ -2,17 +2,17 @@
 # See LICENSE file for licensing details.
 name: github-runner
 display-name: GitHub runner
-description: |
+description: >
   Creates a cluster of self-hosted github runners
-summary: |
-  This charm creates self-hosted GitHub runners. Each unit of this charm will start
-  a configurable number of LXD based containers and virtual machines to host GitHub
-  runners. Each runner performs only one job, after which it unregisters from GitHub
-  to ensure that each job runs in a clean environment.
+summary: >
+  This charm creates self-hosted GitHub runners. Each unit of this charm will start a configurable
+  number of LXD based containers and virtual machines to host GitHub runners. Each runner performs
```

Comment:
"to host GitHub runners" -> "to host them"

Reply:
Fixed